packages/grafana-toolkit/src/cli/index.ts

```diff
@@ -219,6 +220,11 @@ export const run = (includeInternalScripts = false) => {
       await execTask(manifestTask)({ folder: process.cwd() });
     });
 
+  // React generator
+  program.command('component:create').action(async () => {
+    await execTask(componentCreateTask)({});
+  });
```

Comment:
Think this should be treated as internal task

Reply:
How can I make it an internal task?

Reply:
Put it within this condition: 
https://github.com/grafana/grafana/blob/e4b9e0cccd9e5fe335262555a19a4ba782e782cf/packages/grafana-toolkit/src/cli/index.ts#L24